app/views/accounts/_webauthn.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class='clearfix border-top border-blue-light'>
+  <div class='p2 col col-12'>
+    <div class='col col-6 bold'>
+      <%= t('account.index.webauthn') %>
+    </div>
+    <div class='right-align col col-6'>
+      <div class='btn btn-account-action rounded-lg bg-light-blue'>
+        <%= link_to t('account.index.webauthn_add'), webauthn_setup_path %>
+      </div>
+    </div>
+  </div>
+</div>
+<% MfaContext.new(current_user).webauthn_configurations.each do |cfg| %>
+  <div class='p2 col col-12 border-top border-blue-light account-list-item'>
+    <div class='col col-8 sm-6 truncate'>
+      <%= cfg.name %>
+    </div>
+    <% if MfaPolicy.new(current_user).multiple_factors_enabled? %>
```

Comment:
The only change here is I went with a conditional here instead of a `link_to_if`.